src/examples/src/config.tsx

```diff
@@ -302,8 +302,8 @@ export const config = {
 	name: '@dojo/widgets',
 	home: 'src/examples/README.md',
 	themes: [
-		{ label: 'dojo', theme: dojoTheme },
 		{ label: 'material', theme: materialTheme },
+		{ label: 'dojo', theme: dojoTheme },
```

Comment:
Was this reorder intentionally committed? I dug into the commits because I was afraid I had done it 😅

Reply:
Weird I didn't get a notification for this comment! I did change it on purpose, an executive decision because i was always looking at material over dojo in the majority of cases